cmd/buffer-api/main.go

```diff
@@ -25,11 +26,18 @@ func main() {
 	httpPortF := flag.String("http-port", "8000", "HTTP port")
 	debugF := flag.Bool("debug", false, "Enable debug log level.")
 	debugHTTPF := flag.Bool("debug-http", false, "Log HTTP client request and response bodies.")
+	cpuProf := flag.String("cpuprofile", "", "write cpu profile to `file`")
```

Comment:
Jen taková blbost, ale ostatní flagy oddělují slova pomlčkou, tak by to bylo dobrý zachovat i tady. 🙂

Reply:
:+1: